pkg/apply/applier_test.go

```diff
@@ -1943,7 +1943,7 @@ func TestApplierCancel(t *testing.T) {
 
 func TestReadAndPrepareObjectsNilInv(t *testing.T) {
 	applier := Applier{}
-	_, _, err := applier.prepareObjects(nil, object.UnstructuredSet{}, ApplierOptions{})
+	_, _, err := applier.prepareObjects(context.TODO(), nil, object.UnstructuredSet{}, ApplierOptions{})
```

Comment:
t.Context()?

Reply:
done

Reply:
Looks like the test environment doesn't have a new enough Go version to compile Testing.Context. Can you update that?

Reply:
https://github.com/kubernetes/test-infra/pull/34378